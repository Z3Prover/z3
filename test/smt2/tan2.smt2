
; Copyright (c) 2015 Microsoft Corporation

(set-option :numeral-as-real true)
(set-option :pp.decimal true)
(declare-const x Real)
(declare-const y Real)
(declare-const a Int)
(declare-const b Int)

(simplify (tan (+ x (* pi (to_real a)) y)))

(simplify (tan (+ x (* pi 3 (to_real a)) y)))

(simplify (tan (+ x (* pi 3 (to_real a)) y (* (- 5) pi (to_real b)))))

(simplify (tan (+ x pi)))

(simplify (tan (- x pi)))

(simplify (tan (+ x (* 2 pi))))

(simplify (tan (- x (* 2 pi))))

(simplify (tan (+ x (/ pi 2))))
