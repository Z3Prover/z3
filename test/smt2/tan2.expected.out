(tan (+ x y))
(tan (+ x y))
(tan (+ x y))
(tan x)
(tan x)
(tan x)
(tan x)
(tan (+ x (* 0.5 pi)))
